src/data/markdown/translated-guides/es/04 Results visualization/09 StatsD.md

```diff
@@ -21,3 +21,5 @@ Se pueden configurar las siguientes opciones:
 | `K6_STATSD_NAMESPACE`     | El espacio de nombres utilizado como prefijo para todos los nombres de las métricas. El valor por defecto es `k6`.                    |
 | `K6_STATSD_PUSH_INTERVAL` | Configure la frecuencia con la que se envían los datos. El valor por defecto es `1s`.                     |
 | `K6_STATSD_BUFFER_SIZE`   | El tamaño del buffer. El valor por defecto es `20`.                                                            |
+| `K6_STATSD_ENABLE_TAGS`   | Si es `true` habilita el envío de tags. Es `false` por defecto ya que versiones de statsd anterior a v0.9.0 no soportan tags. (disponible desde v0.32.0) |
+| `K6_STATSD_TAG_BLOCKLIST` | Una lista de tags separada por coma que NO deben ser enviados a statsd. Por ejemplo, "tag1,tag2". El valor por defecto es `vu,iter,url`. (disponible desde v0.32.0) |
```

Comment:
@ppcano Is it OK to call these "tags" or should it be something silly like "etiquetas"? :smile:

Reply:
It is ok.